### PR TITLE
feat: use plugins repo, and enable accessibility checker and powerpaste

### PR DESCRIPTION
**Pull request by kenclary**
feat: use plugins repo, and enable accessibility checker and powerpaste

### CODE REVIEW

File: src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/AnswerOption.test.jsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { formatMessage } from '../../../../../../testUtils';
 import { selectors } from '../../../../../data/redux';
 import { AnswerOption, mapStateToProps } from './AnswerOption';
 
+jest.mock('frontend-components-tinymce-advanced-plugins', () => {});
```
Comment (connorhaugh) [inline src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/AnswerOption.test.jsx:7]: You should put these in setuptest.js

**Review by connorhaugh — CHANGES_REQUESTED** [review]
We should consolidate the test statements in setuptest.js.

Also screenshots showing this works would improve my confidence in this pr.

What was the major direction change that led here?

**Comment by kenclary** [comment]
> What was the major direction change that led here?

From a slack thread discussion: "I suspect this means that you can basically override the version of any dependency you would like, no matter how deep it is in the resolved package-lock.json"

Essentially, instead of importing the plugins repo(s) directly in the MFE, and trying to pass the configuration down into the FLCC component(s), just doing it directly in FLCC. In theory, the NPM aliasing (to be) set up for the MFEs will recurse down the dependencies.

**Review by KristinAoki — APPROVED** [review]
Everything appears to work. However when I add
```
 { moduleName: 'frontend-components-tinymce-advanced-plugins', dir: '../frontend-components-tinymce-advanced-plugins-edx' },
```
to my `module.config,js` file in `course-authoring`, missing import errors occur for the plugins. I think this is an issue the `frontend-components-tinymce-advanced-plugins-edx` because it throws an error when I try to run `npm run build`. This will be confirmed when used on stage.

**Comment by kenclary** [comment]
> I think this is an issue the `frontend-components-tinymce-advanced-plugins-edx` because it throws an error when I try to run `npm run build`.

What error do you get? It builds fine on tag/release, and builds fine locally for me.

**Comment by KristinAoki** [comment]
> > I think this is an issue the `frontend-components-tinymce-advanced-plugins-edx` because it throws an error when I try to run `npm run build`.
> 
> What error do you get? It builds fine on tag/release, and builds fine locally for me.

When I run `npm run build` in `frontend-components-tinymce-advanced-plugins-edx` it says that it is unable to build. The errors for when I put it in the `module.config.js` it says that unable to find the export form `tinymce/plugins/{plugin_name)`. But this only occurs when I am using the `module.config.js` override, so this is referencing my local version of `frontend-components-tinymce-advanced-plugins-edx`. I think everything should be fine when referencing the tag/release.

**Comment by kenclary** [comment]
> > > I think this is an issue the `frontend-components-tinymce-advanced-plugins-edx` because it throws an error when I try to run `npm run build`.
> > 
> > What error do you get? It builds fine on tag/release, and builds fine locally for me.
> 
> When I run `npm run build` in `frontend-components-tinymce-advanced-plugins-edx` it says that it is unable to build. The errors for when I put it in the `module.config.js` it says that unable to find the export form `tinymce/plugins/{plugin_name)`. But this only occurs when I am using the `module.config.js` override, so this is referencing my local version of `frontend-components-tinymce-advanced-plugins-edx`. I think everything should be fine when referencing the tag/release.

OK. I suspect you could fix your local build by pulling the latest code from main (and maybe deleting `node_modules/` and running `npm i` and such).

In the meantime, I'll go ahead and merge this so it can make it into the MFEs.

**Pull request merged**